locales/de.json

```diff
@@ -774,38 +774,38 @@
   "app.tos.trading.listings.logIn": "Wenn du für 7 Tage inaktiv bist werden deine Angebote automatisch deaktiviert.{br}Die Angebote bleiben bestehen - du kannst sie reaktivieren, indem du sie auf 'sichtbar' stellst.",
-  "app.tos.trading.listings.listingQuantities": "Du kannst bis zu 3 sichtbare Angebote mit unterschiedlichen Preisen für denselben Gegenstand erstellen.{br}Angebotene Anzahl",
+  "app.tos.trading.listings.listingQuantities": "Du kannst bis zu 3 sichtbare Angebote mit unterschiedlichen Preisen für denselben Gegenstand erstellen.",
   "app.tos.trading.listings.trade": "Du musst den Tausch ausführen können. Es ist nicht erlaubt Angebote für Gegenstände zu erstellen die du nicht besitzt.",
-  "app.tos.trading.listings.noRequirement": "You cannot require users to buy, sell, or trade additional items or perform additional tasks prior to trading your listed item.",
-  "app.tos.trading.listings.refusing": "Du darfst ein von dir gemachtes Angebot nicht verweigern.",
-  "app.tos.trading.listings.refusing.listingQuantities": "Quantity only indicates how many you are willing to buy or sell.{br}You cannot refuse to buy or sell any amount at or lower than your listed quantity.",
+  "app.tos.trading.listings.noRequirement": "Du darfst von Benutzern nicht verlangen zusätzliche Items zu kaufen, verkaufen oder handeln oder vor dem Handel zusätzliche Aufgaben zu erfüllen.",
+  "app.tos.trading.listings.refusing": "Du darfst ein von dir gemachtes Angebot nicht verweigern. Du bist an den angegeben Preis zum Zeitpunkt der Kontaktaufnahme des Käufers gebunden. Die einzigen Ausnahmen sind wenn dein Status auf “Unsichtbar“ gesetzt ist (s. 1.2.3.) oder offensichtliche Fehler wie zum Beispiel die Eingabe eines hohen Preises und niedriger Anzahl in das falsche Feld. In diesem Fall solltest du dein Angebot innerhalb von 5 Minuten korrigieren.",
+  "app.tos.trading.listings.refusing.listingQuantities": "Die Menge eines Angebotes gibt nur an, wie viel du zu kaufen/verkaufen bereit bist. Du kannst keinen Handel verweigern, der innerhalb der von dir angegebenen Menge liegt.",
   "app.tos.trading.process": "Handel",
-  "app.tos.trading.process.definition": "The trading process is the time between the initial message of the “Contactor” (which should be the standard warframe.market copy and paste message) and the trade fulfillment of the “Lister”.{br}The trading process may have varied time limitations based on the “Lister's” online status on the website.",
+  "app.tos.trading.process.definition": "Handel bezeichnet den Zeitraum zwischen der ersten Nachricht des Kontaktierenden (welche die Standard warframe.market Copy-and-Paste Nachricht sein sollte) und der Erfüllung des Handels durch den Anbietenden. Die zeitlichen Einschränkungen des Handels hängen vom Online-Status des Anbietenden ab (s. 1.2.1f)",
```

Comment:
you had removed the {br} here, which is to indicate a line break

Reply:
Fixed it.